src/main/kotlin/fi/metatavu/metaform/server/controllers/ReplyController.kt

```diff
@@ -361,8 +361,14 @@ class ReplyController {
             includeRevisions: Boolean,
             fieldFilters: FieldFilters? = null,
             firstResult: Int? = null,
-            maxResults: Int? = null
+            maxResults: Int? = null,
+            orderBy: ReplyOrderCriteria? = null,
+            latestFirst: Boolean? = null
     ): List<Reply> {
+        // mimic the original behavior
+        val _orderBy = orderBy ?: ReplyOrderCriteria.CREATED
+        val _latestFirst = latestFirst ?: false
```

Comment:
Use different variable names since _ prefix usually means that these variables are unused